Dockerfile

```diff
@@ -66,11 +66,9 @@ EXPOSE 8000
 EXPOSE 5901
 
 RUN echo "mycontainer" > /etc/hostname
-RUN echo "127.0.0.1	localhost" > /etc/hosts
-RUN echo "127.0.0.1	mycontainer" >> /etc/hosts
 
-RUN addgroup --gid $GROUP_ID enhydris
-RUN adduser --uid $USER_ID --gid $GROUP_ID --disabled-password --gecos "" enhydris
+RUN groupadd -o --gid $GROUP_ID enhydris
```

Comment:
Why is this? What is the problem with addgroup?

Reply:
I was constantly getting this error and the solution here solved it for me. But I don't clearly understand why it should solve it though. 
 https://github.com/mxrch/GHunt/issues/62

Reply:
`./build.sh` tries to build the container in such a way that the user id of user "enhydris" and group id of group "enhydris" are the same as the user id and group id of the user who is running `./build.sh` on the host. This will make it easier to share files between the docker host and the docker container.

`./build.sh` uses the commands `id -u` and `id -g` to get the user and group id. Either these commands are inappropriate on the Mac, or the user and group ids are strange. Could you find out?

Reply:
Okay, I will report back once I have something concrete.